docs/workloads/azure/data/components/infrastructure/core_data_platform_azure.md

```diff
@@ -10,28 +10,84 @@ keywords:
   - resource group
   - adf
   - adls
+  - blob
   - databricks
   - key vault
 ---
 
-## TODO
+Azure DevOps pipelines for building and deploying the core infrastructure are configured in
+[build](https://github.com/amido/stacks-azure-data/tree/main/build) folder.
 
-Documentation on the Infrastructure used and (maybe) the high level deployment approach.
-Deployment details (usage manual, CLI commands) should be captured in
-[getting_started](../../getting_started/core_data_platform_deployment_azure.md).
+Core data platform resources are defined within Terraform templates and grouped inside
+[deploy/azure](https://github.com/amido/stacks-azure-data/tree/main/deploy/azure) directory.
+There are two subfolders in this directory:
 
-This should be an extension to [intro_data_azure.md](../../intro_data_azure.md).
+* `infra`
+* `networking`
 
-### Infrastructure deployed
+## Infra
 
-* Resource Group
-* Azure Data Factory
-* Azure Data Lake Storage Gen2
-* Azure Blob Storage (for config files)
-* Azure Key Vault
-* Log Analytics Workspace
-* Databricks Workspace
-    * Azure Key Vault-backed secret scope
+`infra` subfolder contains the following definitions:
 
+1. **Resource Group**
+2. **MS SQL Database** instance with database schemas
+3. **Key Vault**
+4. **Key Vault Secrets**:
+    * Connection strings for the created SQL databases
+    * Passwords to the SQL Databases
+    * Service Principal Secret
```

Comment:
We should have it. It is in KV and it is used. I added all the secrets that we are using, even if they are currently missing from Terraform resources, as I thought it will be easy to forget to add them in two documentation repositories when they are finally done.